.travis.yml

```diff
@@ -33,15 +33,10 @@ matrix:
           packages:
             - g++-5
     - os: linux
-      compiler: clang++-3.7
-      env: AUTOTOOLS=no COVERAGE=no BUILD=static
-      addons:
-        apt:
-          sources:
-            - ubuntu-toolchain-r-test
-            - llvm-toolchain-precise-3.7
-          packages:
-            - clang-3.7
+      compiler: clang
```

Comment:
Looks like this would regress to clang 3.5.0 - https://docs.travis-ci.com/user/languages/cpp/#clang

Reply:
The documentation is out of date, clang on Travis is actually v5.0 (see the build log and multiple issues on the travis-ci repo about the docs being out of date).

Travis docs on almost every subject are out of date.

Reply:
You're right. Apologies.